Gems/SceneProcessing/Code/Source/Config/Components/SceneProcessingConfigSystemComponent.cpp

```diff
@@ -139,6 +149,68 @@ namespace AZ
             return success;
         }
 
+        void SceneProcessingConfigSystemComponent::LoadScriptSettings()
+        {
+            auto* registry = AZ::SettingsRegistry::Get();
+            AZ_Assert(registry, "AZ::SettingsRegistryInterface should already be active!");
```

Comment:
Can you make these assert messages a bit more actionable? If a content creator hits this, they won't know what to do.

Reply:
Saddly not really. If the SettingsRegistryInterface is missing, most (all?) of the component applications would be broken. (Basically this would be a fix AzCore).

The best we can do here is early out + an error if the assert is not good enough.

Reply:
Maybe in that case the message could be something like "AZ::SettingsRegistry interface was not available. This is a fatal code issue."?